src/script/sign.js

```diff
@@ -0,0 +1,83 @@
+const email_input = document.querySelector("#signin_email");
+const password_input = document.querySelector("#signin_password");
+const error_email = document.querySelector("#error_email");
+const error_password = document.querySelector("#error_password");
+const eyes = document.querySelector(".eye_button");
+
+// 이메일 검사
+function validateEmail() {
+    // 이메일 값이 없는 경우
+    if (email_input.value.trim() === "") {
```

Comment:
간단한 조건이라 이렇게 작성하셔도 좋지만 변수로 빼서 작업하시면 이해하기 더 편할 것 같아요.
```suggestion
const isEmpty = email_input.value.trim() === ""
    if (isEmpty) {
```